pack/deb.mk

```diff
@@ -34,9 +34,6 @@ PREBUILD := prebuild.sh
 PREBUILD_OS := prebuild-$(OS).sh
 PREBUILD_OS_DIST := prebuild-$(OS)-$(DIST).sh
 
-# gh-7: Ubuntu/Debian should export DEBIAN_FRONTEND=noninteractive
-export DEBIAN_FRONTEND=noninteractive
```

Comment:
We shouldn't delete it for other targets. For example, the following script may fail after this change: https://github.com/tarantool/queue/blob/addd1387c9e5677040c3b1f0c1af88303137306a/debian/prebuild.sh.

Reply:
Fixed.